### PR TITLE
Rerun examples and add aio deploy_local/remote

### PR DESCRIPTION
**Pull request by cliveseldon**
* Reruns examples and recreated markdown with cell contents
 * Add conda.yaml files to git where needed
 * Update asyncio for new options and deply local/remote

### CODE REVIEW

File: docs/examples/kfserving/README.ipynb [hunk 1]
```diff
@@ -45,12 +45,35 @@
   },
```
Comment (axsaucedo) [inline docs/examples/kfserving/README.ipynb:45]: <pre class="ql-syntax" spellcheck="false">Line #5.                "runtime": "tempo.kfserving.KFServingKubernetesRuntime",
</pre><p>In this case you would probably be using <code>from tempo.serve.metadata import KFServingOptions</code>  so you don't ahve to specify the underlying runtime in <code>remote_options</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/tempo/pull/178/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>

File: docs/examples/mab-thompson-sampling-tempo/README.ipynb [hunk 2]
```diff
@@ -9,17 +9,9 @@
   },
```
Comment (axsaucedo) [inline docs/examples/mab-thompson-sampling-tempo/README.ipynb:9]: <p>Would we want to remove all the outputs from this notebook?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/tempo/pull/178/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

**Comment by axsaucedo** [comment]
Looks good, just added two comments, as may be worth leaving the outputs for the MAB notebook, as well as the KFServingOptions variable

File: docs/examples/mab-thompson-sampling-tempo/README.ipynb (hunk 2 above)
Comment (cliveseldon) [reply docs/examples/mab-thompson-sampling-tempo/README.ipynb:9]: Yeah I feel maybe this notebook needs to be moved to incubating or updated with more docs?

Comment (axsaucedo) [reply docs/examples/mab-thompson-sampling-tempo/README.ipynb:9]: Ok makes sense i can update with more docs

**Comment by axsaucedo** [comment]
Merging - we can update the kfserving options separate

**Pull request merged**